src/containers/ImageUploader/CreateImage.tsx

```diff
@@ -36,7 +35,7 @@ const CreateImage = ({ history, isNewlyCreated, showSaved }: Props & RouteCompon
     setLicenses(license);
   };
 
-  const onCreateImage = async (imageMetadata: NewImageMetadata, image: string | Blob) => {
+  const onCreateImage = async (imageMetadata: UpdatedImageMetadata, image: string | Blob) => {
```

Comment:
Nå er det litt missmatch mellom navngivinga her, skal funksjonen oppdatere et bilde eller lage et nytt bilde? For funksjonsnavnet sier `create` men prop-type sier `updated`

Reply:
Yep, men den passes inn i `onUpdate` funksjonen i `ImageForm` som er den samme som oppdateres med.
Kan se om jeg kan hacke meg rundt det uten å skrive om for mye. Men om det blir for ille så stemmer jeg for at vi lar det gå.

Reply:
Typene ble ganske kompitable når jeg endret de til å ha id som var `number` så kunne trikse litt :+1: